source/conf.py

```diff
@@ -27,7 +27,7 @@
 # The short X.Y version
 version = u''
 # The full version, including alpha/beta/rc tags
-release = u'1.0.0'
+release = u'1.0.1'
```

Comment:
minor, not really necessary. If it messes with linking, I wou;d just leave it as 1.0.0